config/environments/development.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  config.webpacker.check_yarn_integrity = false
```

Comment:
Can we also make this change in `config/environments/test.rb`?

Reply:
Can do!

Reply:
In: d86bb5e